packages/kotti-ui/source/kotti-form/tests.ts

```diff
@@ -139,7 +97,7 @@ describe('KtForm', () => {
 		it('should deepClone values before passing it to field, to prevent deep mutation madness', () => {
 			const DEEP_VALUE_REFERENCE = { deep: true }
 
-			const wrapper = mount(TestForm2, {
```

Comment:
This is also wrong, as this test is testing deep clone behavior as well

Reply:
same as https://github.com/3YOURMIND/kotti/pull/600#discussion_r809985330